src/docs/testing/integration-tests/index.md

```diff
@@ -0,0 +1,290 @@
+---
+title: Integration testing
+description: Learn how to write integration tests
+---
+
+This page describes how to use the [integration_test][] package to run
+integration tests. Tests written using this package have the following
+properties:
+ 
+* Compatibility with the `flutter drive` command, for running tests on a
+physical device or emulator.
+* The ability to be run on [Firebase Test Lab][], enabling automated testing on a
+variety of devices.
+* Compatibility with [flutter_test][] APIs, enabling tests to be written in a
+similar style as [widget tests][]
+
+## Overview
+
+There are three types of tests that Flutter supports. A **unit** test verifies
+the behavior of a method or class. A **widget test** verifies the behavior of
+Flutter widgets without running the app itself. An **integration test** runs the
+full app.
+
+During development, you are probably writing the code on a
+desktop computer, called the **host** machine, and running the app on a mobile
+device, browser, or desktop application, called the **target** device. (If you
+are using a web browser or desktop application, the host machine is also the
+target device.)
+
+The flutter_driver package runs integration tests written in Dart on a target
+device and reports the results to the host.
+
+The [integration_test][] package lets you run tests directly on the target
+device. This allows you to run integration tests on multiple Android or iOS
+devices using Firebase Test Lab. 
+
+These tests can also be run using `flutter_driver`. This allows you to run
+integration tests on a target device or emulator and view the results on your
+host machine. Previously, `flutter_driver` APIs needed to be used to perform
+actions a user would, like tapping on a button. But with `integration_test`, any
+action supported by the `flutter_test` package can be used. This makes writing
+integration tests more like writing [widget tests][].
```

Comment:
I added this paragraph to explain how flutter_driver, and integration_test relate to each other, and why integration_test is more familiar to those who have written a widget test before. LMWYT.